unabridged-works-volume-3/original/12-jews-natural-spiritual.adoc

```diff
@@ -977,11 +1032,16 @@ And the Maintenance thereof;
 some few Questions answered in Truth and Plainness of heart,
 and left to the Witness and Testimony of God in other Men`'s Consciences.
 
-Question 1. WHICH is the true church,
+[.numbered-group]
```

Comment:
Hmmmm.... because of the way it's a question x / answer back and forth below, I think I would feel a little better making these _all_ discourse-parts, instead of interleaving DP and numbered.